packages/components/src/form-token-field/suggestions-list.js

```diff
@@ -105,6 +105,31 @@ class SuggestionsList extends Component {
 					);
 
 					/* eslint-disable jsx-a11y/click-events-have-key-events */
+
+					    var label = suggestion.label;
+
+					    var decodeEntities = (function() {
```

Comment:
Hey there! thanks for the PR. That function is already available in the html-entities package.

```
import { decodeEntities } from '@wordpress/html-entities';
```

That said, it seems instead of doing this here in the generic component, we should probably do it when using the component, so somewhere in the  `PageAttributesParent` component.

Reply:
Hello youknowriad) Thanks for help ) I changed my commit according to your comment )

Reply:
It seems not all tests are passed , can you explain please what I must to do to pass all tests ?